spikesorters/herdingspikes/herdingspikes.py

```diff
@@ -119,50 +207,3 @@ def _run(self, recording, output_folder):
     @staticmethod
     def get_result_from_folder(output_folder):
         return se.HS2SortingExtractor(file_path=Path(output_folder) / 'HS2_sorted.hdf5', load_unit_info=True)
-
-
-HerdingspikesSorter._default_params = {
-    # core params
```

Comment:
I think I preferd this way thanb then nested as you did.

Reply:
why? it is like this for all other sorters and it's the same in the end , no?

Reply:
yes but when the params list is very long.
it breaks the readablity of the code.
as you want it do not matter a lot.